client/src/utils/wagmi.ts

```diff
@@ -10,14 +16,64 @@ const infuraId = process.env.INFURA_ID;
 
 const chainsAvailable: Chain[] = [];
 
+const fantomTestnet: Chain = {
+  id: 4002,
+  name: "Fantom Testnet",
+  network: "fantom testnet",
+  iconUrl: "https://fantomrock.com/img/tokens/0.png",
+  nativeCurrency: {
+    decimals: 18,
+    name: "Fantom",
+    symbol: "FTM",
+  },
+  rpcUrls: {
+    default: "https://rpc.testnet.fantom.network/",
+  },
+  blockExplorers: {
+    default: { name: "ftmscan", url: "https://testnet.ftmscan.com" },
+  },
+  testnet: true,
+};
+
+const fantomMainnet: Chain = {
+  id: 250,
+  name: "Fantom",
+  network: "fantom mainnet",
+  iconUrl:
+    "https://ipfs.io/ipns/k51qzi5uqu5di9fpr1u08putyfwqdzwue8dbxq0047mid8hj85hfetm9bw71tv",
+  nativeCurrency: {
+    decimals: 18,
+    name: "Fantom",
+    symbol: "FTM",
+  },
+  rpcUrls: {
+    default: "https://rpc.ankr.com/fantom/",
+  },
+  blockExplorers: {
+    default: { name: "ftmscan", url: "https://ftmscan.com" },
+  },
+  testnet: false,
+};
+
 if (process.env.REACT_APP_LOCALCHAIN) {
   chainsAvailable.push(chain.hardhat);
 }
 
 if (process.env.REACT_APP_ENV === "production") {
-  chainsAvailable.push(chain.optimism, chain.goerli, chain.optimismKovan);
+  chainsAvailable.push(
```

Comment:
shouldn't we put only fantom in production?

Reply:
this was per @michellema1208 request